test/acceptance/ApiSubscriptionFieldsSpec.scala

```diff
@@ -154,13 +153,13 @@ class ApiSubscriptionFieldsSpec extends AcceptanceTestSpec
       val sfr = contentAsJson(resultFuture).validate[BulkSubscriptionFieldsResponse]
 
       sfr.isSuccess shouldBe true
-      sfr.get should matchPattern { case BulkSubscriptionFieldsResponse(Seq(SubscriptionFieldsResponse(FakeRawIdentifier, _, SampleFields1))) => }
+      sfr.get should matchPattern { case BulkSubscriptionFieldsResponse(Seq(SubscriptionFieldsResponse(`fakeRawClientId`, "acontext", "1.0.2", _, SampleFields1))) => }
     }
 
-    scenario("the API is called to store more values for an existing subscription identifier") {
+    scenario("the API is called to store more values for an existing subscription field") {
```

Comment:
`the API is called to update existing subscription fields`

Reply:
Done